docs/platform/concepts/managed-users.rst

```diff
@@ -0,0 +1,10 @@
+Managed users 
+==============
+
+The managed users feature provides a centralized way of managing all of your organization's users, including editing their profiles, resetting passwords, and :doc:`setting authentication policies </docs/platform/howto/set-authentication-policies>`.
+
+When you :doc:`verify a domain </docs/platform/howto/manage-domains>`, existing organization users automatically become managed users. New users can be provisioned and managed through your identity provider (IdP). 
+
+A managed user cannot create new organizations unless they are a super admin of the organization that they are managed by. They are also not allowed to edit their user profiles.
```

Comment:
Remove "They are also not allowed to edit their user profiles." as they are allowed unless they are managed by SCIM but this is another feature.